lib/standalone.cjs

```diff
@@ -108,6 +99,17 @@ async function standalone({
 		quietDeprecationWarnings,
 	});
 
+	const configForFile = await getConfigForFile(stylelint, configFile, configFile);
```

Comment:
If a config is passed through the cli it should be used instead.
i.e. `config ?? […]`

Reply:
I think the requested changes to the getFormatterFunction resolved this too.